apps/docs/src/app/platform/component-docs/platform-menu/platform-menu-examples/platform-menu-with-icons-example.component.scss

```diff
@@ -0,0 +1,10 @@
+@import "~@angular/cdk/overlay-prebuilt.css";
+@import '~fundamental-styles/dist/menu';
+
+fdp-button {
+    margin-right: 1rem;
+}
+
+fd-image {
```

Comment:
Query: Is it expected customer should also add this class properties.

Reply:
@sKudum that's just to make it more pretty in example